server/public/model/preference.go

```diff
@@ -97,6 +100,13 @@ func (o *Preference) IsValid() *AppError {
 		}
 	}
 
+	if o.Category == PreferenceCategorySidebarSettings && o.Name == PreferenceLimitVisibleDmsGms {
+		visibleDmsGmsValue, convErr := strconv.Atoi(o.Value)
+		if convErr != nil || visibleDmsGmsValue < 1 || visibleDmsGmsValue > PreferenceMaxLimitVisibleDmsGmsValue {
+			return NewAppError("Preference.IsValid", "model.preference.is_valid.limit_visible_dms_gms.app_error", nil, "value="+o.Value, http.StatusBadRequest)
```

Comment:
I don't see a translation string for "model.preference.is_valid.limit_visible_dms_gms.app_error". How is the CI passing?

Reply:
You have to add the string in your en.json file. Look at the failure here: https://github.com/mattermost/enterprise/actions/runs/9879507441/job/27285915209